services/fis/tests_fis/fixtures/joint.py

```diff
@@ -66,35 +68,38 @@ class JointFixture:
 
 
 @pytest_asyncio.fixture
-async def joint_fixture(kafka: KafkaFixture) -> AsyncGenerator[JointFixture, None]:
+async def joint_fixture(
+    kafka: KafkaFixture, mongodb: MongoDbFixture
+) -> AsyncGenerator[JointFixture, None]:
     """Generate keypair for testing and setup container with updated config"""
     keypair = generate_key_pair()
     private_key = encode_key(key=keypair.private)
 
     token, token_hash = generate_token_and_hash()
 
-    config = get_config(sources=[kafka.config])
+    config = get_config(sources=[kafka.config, mongodb.config])
     # cannot update inplace, copy and update instead
     config = config.model_copy(
         update={"private_key": private_key, "token_hashes": [token_hash]}
     )
-    async with prepare_core(config=config) as (
-        upload_metadata_processor,
-        legacy_upload_metadata_processor,
-    ):
-        async with prepare_rest_app(
+    async with (
+        prepare_core(config=config) as (
+            upload_metadata_processor,
+            legacy_upload_metadata_processor,
+        ),
+        prepare_rest_app(
             config=config,
             core_override=(upload_metadata_processor, legacy_upload_metadata_processor),
-        ) as app:
-            async with AsyncTestClient(app=app) as rest_client:
-                yield JointFixture(
-                    config=config,
-                    keypair=keypair,
-                    payload=TEST_PAYLOAD,
-                    token=token,
-                    kafka=kafka,
-                    rest_client=rest_client,
-                    s3_endpoint_alias=config.selected_storage_alias,
-                    upload_metadata_processor=upload_metadata_processor,
-                    legacy_upload_metadata_processor=legacy_upload_metadata_processor,
-                )
+        ) as app,
+    ):
+        async with AsyncTestClient(app=app) as rest_client:
```

Comment:
why don't you use the new syntax here as well?

Reply:
It is so ([5c19d57](https://github.com/ghga-de/file-services-backend/pull/28/commits/5c19d57bda7d294a16f09373d32619cb4d1a0261))